lib/new_relic/agent/instrumentation/action_controller_subscriber.rb

```diff
@@ -16,6 +16,7 @@ def start(name, id, payload) # THREAD_LOCAL_ACCESS
           controller_class = controller_class(payload)
 
           if state.is_execution_traced? && !should_ignore(payload, controller_class)
+            # TODO: MLT - possible Rails entrypoint for mlt
```

Comment:
This could be a good entry point specifically for ActionController. We will likely need to add something like this to each of the `lib/new_relic/agent/*_subscriber.rb` files to get full Rails support.

This also reminds me of some [pre-spike work](https://docs.google.com/document/d/19NkLomuuCjvOudnqu2FeeWNeTqEhRqNxI-6oOzj09j4/edit?usp=sharing) (Internal) we discussed in the fall for ActionPack. We were thinking about rolling our ActionController instrumentation up into ActionPack as a whole to use the actionpack gem as the top-level reference.

Reply:
I took a pass at satisfying the TODO here with #1085 by updating `NewRelic::Agent::Instrumentation::ActionControllerSubscriber#start_transaction_or_segment`. By the time that method is called, we have access to the controller class and method names that we need.

We should be able to do something similar for the other subscriber classes.

Reply:
As discussed, we don't currently hang method tracers on any other Rails methods via subscriber messages being parsed, so we won't start doing so at this time just for the sake of MLT. We do, however, explicitly look for ActiveJob and set method tracers there, and those tracers will automatically have code metrics now.